packages/css/src/components/icon-button/README.md

```diff
@@ -2,14 +2,15 @@
 
 # Icon Button
 
-A button containing only an icon.
+An icon in a button that can be aligned with text.
 
 ## Guidelines
 
-- Use this button only if the icon is entirely clear and understandable.
-  Many icons are interpreted differently by different users.
-  Only use universally recognized icons, such as a close icon or a play icon.
-- Provide a label text that describes what the button does for screen reader users.
+- Choose an Icon Button instead of a regular [Button](https://designsystem.amsterdam/?path=/docs/components-buttons-button--docs) if it must align with text.
+- Match the size of the Icon Button to that text.
+- Use only universally recognised [icons](http://designsystem.amsterdam/?path=/docs/brand-assets-icons--docs).
```

Comment:
We don't currently explain what the universally recognised icons are, so this guideline is a little ambiguous.

Reply:
Hm, I'm not sure about this. We're changing the docs from 'universally recognized icons, such as close or play' to 'be aware'. We going from specific(-ish) to vague. Maybe we should not change this guideline until we have a list of icons we consider universally recognized?